blockchain-scrapers/blockchains/bitcoin-cash/docker-compose.bitcoin-cash.yml

```diff
@@ -26,7 +26,7 @@ services:
     build:
       context: $GOPATH
       dockerfile: $GOPATH/src/github.com/diadata-org/api-golang/blockchain-scrapers/blockchains/bitcoin-cash/node/Dockerfile-bch
-    image: ${DOCKER_HUB_LOGIN}/blockchain-scrapers_bch
+    image: ${DOCKER_HUB_LOGIN}/blockchain-scrapers_bch:latest
     depends_on:
```

Comment:
could you replace with 
${DOCKER_HUB_LOGIN}/{STACKNAME}_bch
in all docker-compose files ?
not sure if the :latest is needed btw